tests/test_client.py

```diff
@@ -346,6 +346,52 @@ def test_get_decisions(self):
             assert(response.is_ok())
             assert(response.body['data'][0]['id'] == 'block_user')
 
+    def test_get_decisions_entity_session(self):
+        mock_response = mock.Mock()
+        get_decisions_response_json =  \
+            '{' \
+                '"data": [' \
+                    '{' \
+                        '"id": "block_user",' \
```

Comment:
well, it's a very minor nit, but `id`, `name`, and `description` do not match the entity type, though it isn't really related to what the test case is supposed to test.